# Collaborative genetic studies of idiopathic pulmonary fibrosis
Idiopathic pulmonary fibrosis (IPF) is characterized by the build-up of scar tissue in the lungs. It is believed that the damage to the alveolar epithelium is followed by an aberrant wound healing response leading to the deposition of dense fibrotic tissue, reducing the lungs’ flexibility and inhibiting gas transfer. IPF still has limited therapeutic interventions and a high mortality rate within 3-5 years from diagnosis.

To date, genome-wide association studies (GWAS) of IPF susceptibility have associated common variants (minor allele frequency [MAF]>5%) near genes involved in host defence, telomere maintenance, cell-cell adhesion and signalling in disease susceptibility. Common variants near MUC5B and TOLLIP genes also show association with survival time after diagnosis of IPF (1,2).

# 3-way meta-GWAS of IPF susceptibility (2020)
Building up on published GWAS results (1,3,4) and novel study samples, we have performed a large GWAS of IPF susceptibility (5) to identify novel genes and further advance in the understanding of IPF pathogenesis and risk. The discovery stage of the study comprised up to 2,668 IPF cases and 8,591 controls and replication was pursued in an additional 1,456 IPF cases and 11,874 controls.

<p align="center">
  <img src="https://github.com/genomicsITER/PFgenetics/blob/master/Figure_susceptibility_GWAS-IPF.png" width="auto"/>
</p>
<p><br></p>
<!-- ![What is this](Figure_susceptibility_GWASIPF.png) -->

**Sample sizes for genome-wide analyses**

## Access to the results of this study
To help advance IPF research and allow the wider research community to access to the most accurate effect sizes for genetic variants on a genome-wide scale, the variant summary data from the discovery stage of the study by Allen et al. (5; https://doi.org/10.1164/rccm.201905-1017OC) (1.31 Gb) can be accessed after an internal assessment of formal requests received.

## What type of results would be made available?
No individual level data will be made available. Granted requests will have access to a file with the GWAS meta-analysis variant summary data with information for the following descriptors:

   •	**rsid**
   
   •	**chromosome_position**
      [According to GRCh37/hg19 reference]
      
   •	**non_effect_allele**
   
   •	**effect_allele**
   
   •	**effect_allele_frequency**
      [Frequency taken from across all three studies included in the meta-analysis]
      
   •	**studies_included**
      [This is a piece of text that is Y if it is in the study and N if it is not. For example if the variant was in all three       studies it would read Y:Y:Y (with UK first, Chicago second and Colorado last), if it was in the UK and Chicago but not         included in Colorado it would read Y:Y:N]
      
   •	**imputation_quality_by_study**
      [The imputation r2 by study in the same order as the studies_included column]
      
   •	**effect_direction_by_study**
      [A piece of text with a plus sign (+) if the effect allele was associated with an increased risk of IPF (i.e. beta > 0)         and a minus sign (–) if associated with a decreased risk of IPF (i.e. beta < 0) by study. For example if the effect           allele had lower risk in the UK study but higher risk in the Chicago and Colorado studies it would read -:+:+]
      
   •	**beta**
      [This is the beta from the meta-analysis]
      
   •	**standard_error**
      [This is the standard error from the meta-analysis]
      
   •	**p**
      [This is the p value from the meta-analysis]

# 5-way meta-GWAS of IPF susceptibility (2021)
To maximise power and to generate more accurate effect size estimates, we performed a genome-wide meta-analysis across all five studies included in the previous IPF risk GWAS (6). We utilised the distribution of effect sizes across the five studies to assess the replicability of the results and identified five robust novel genetic association signals. A total of 4,125 IPF cases, 20,464 controls, and 7,554,248 genetic variants were included in the analysis.

<p align="center">
  <img src="https://github.com/genomicsITER/PFgenetics/blob/master/Figure_susceptibility_GWASIPF-5way.png" width="auto"/>
</p>
<p><br></p>
<!-- ![What is this](Figure_susceptibility_GWASIPF-5way.png) -->

**Sample sizes for genome-wide analyses**

## Access to the results of this study
To help advance IPF research and allow the wider research community to access to the most accurate effect sizes for genetic variants on a genome-wide scale, the variant summary data resulting from this second meta-analysis by Allen et al. (6; https://doi.org/10.1101/2021.12.06.21266509) (roughly 0.8 Gb) can be accessed after an internal assessment of formal requests received.

## What type of results would be made available?
No individual level data will be made available. Granted requests will have access to a file with the GWAS meta-analysis variant summary data with information for the following descriptors:

   •	**chromosome_position**
      [According to GRCh37/hg19 reference]
   
   •	**rsid**
      
   •	**non_effect_allele**
   
   •	**effect_allele**
   
   •	**maf**
      [Minor allele frequency taken from across all five studies included in the meta-analysis]
      
   •	**Studies**
      [Number of studies contributing to the result]
      
   •	**n**
      [This is the sample size. Note, as not all variants were included in all studies, this value varies by SNP]
      
   •	**Direction**
      [Effect direction by study. A piece of text with a plus sign (+) if the effect allele was associated with an increased risk of IPF (i.e. beta > 0) and a minus sign (–) if associated with a decreased risk of IPF (i.e. beta < 0) by study (in the order UK, Colorado, Chicago, UUS and then Genentech). If the variant was not included in the analysis, this is recorded with a question mark (?)]
      
   •	**beta**
      [This is the beta from the meta-analysis]
      
   •	**standard_error**
      [This is the standard error from the meta-analysis]
      
   •	**p**
      [This is the p value from the meta-analysis]

## Who should I contact for requesting access to the data?
Requesters should contact **`GWASIPF[dot]adm[at]gmail.com`** for requesting access to the data. Designated delegates of this collaborative effort will review the requests. Within a few days, an email with the instructions for data downloading will be obtained in response.

## Which information should I submit to access the variant summary data?
Requesters should provide a signed document including the following information: full name and title of the PI, affiliation, and a brief description of the project and the aims where the data is to be used. 

The document should explicitly state that the data requester also agrees to: use the data only as part of that indicated research, not redistribute the data outside the project without permission, acknowledge the provenance of the data in any dissemination of results.

A template document for the request is [here](https://github.com/genomicsITER/PFgenetics/blob/master/Request_template.txt).

## Collaborative group
Dr. Carlos Flores, Research Unit - Hospital Universitario N.S. de Candelaria & Genomics Division – Instituto Tecnológico y de Energías Renovables (ITER), Spain.

Dr. R. Gisli Jenkins, National Institute for Health Research, Nottingham Biomedical Research Centre, Nottingham University Hospitals & Division of Respiratory Medicine – University of Nottingham, UK.

Dr. Imre Noth, Division of Pulmonary & Critical Care Medicine, University of Virginia, USA.

Dr. Justin M. Oldham, Department of Internal Medicine, University of California Davis, USA.

Dr. Louise V. Wain, Department of Health Sciences - University of Leicester & Leicester Respiratory Biomedical Research Centre - Glenfield Hospital, UK.

## References
1: Noth I, Zhang Y, Ma SF, Flores C, Barber M, Huang Y, Broderick SM, Wade MS, Hysi P, Scuirba J, Richards TJ, Juan-Guardela BM, Vij R, Han MK, Martinez FJ, Kossen K, Seiwert SD, Christie JD, Nicolae D, Kaminski N, Garcia JGN. Genetic variants associated with idiopathic pulmonary fibrosis susceptibility and mortality: a genome-wide association study. Lancet Respir Med 2013, 1: 309-317. doi: https://doi.org/10.1016/S2213-2600(13)70045-6. PubMed PMID: 24429156.

2: Peljto AL, Zhang Y, Fingerlin TE, Ma SF, Garcia JG, Richards TJ, Silveira LJ, Lindell KO, Steele MP, Loyd JE, Gibson KF, Seibold MA, Brown KK, Talbert JL, Markin C, Kossen K, Seiwert SD, Murphy E, Noth I, Schwarz MI, Kaminski N, Schwartz DA. Association between the MUC5B promoter polymorphism and survival in patients with idiopathic pulmonary fibrosis. JAMA 2013, 309: 2232-9. doi: https://doi.org/10.1001/jama.2013.5827. PubMed PMID: 23695349.

3: Fingerlin TE, Murphy E, Zhang W, Peljto AL, Brown KK, Steele MP, Loyd JE, Cosgrove GP, Lynch D, Groshong S, Collard HR, Wolters PJ, Bradford WZ, Kossen K, Seiwert SD, du Bois RM, Garcia CK, Devine MS, Gudmundsson G, Isaksson HJ, Kaminski N, Zhang Y, Gibson KF, Lancaster LH, Cogan JD, Mason WR, Maher TM, Molyneaux PL, Wells AU, Moffatt MF, Selman M, Pardo A, Kim DS, Crapo JD, Make BJ, Regan EA, Walek DS, Daniel JJ, Kamatani Y, Zelenika D, Smith K, McKean D, Pedersen BS, Talbert J, Kidd RN, Markin CR, Beckman KB, Lathrop M, Schwarz MI, Schwartz DA. Genome-wide association study identifies multiple susceptibility loci for pulmonary fibrosis. Nat Genet 2013, 45: 613-20. doi: https://doi.org/10.1038/ng.2609. PubMed PMID: 23583980.

4: Allen RJ, Porte J, Braybrooke R, Flores C, Fingerlin TE, Oldham JM, Guillen-Guio B, Ma SF, Okamoto T, John AE, Obeidat M, Yang IV, Henry A, Hubbard RB, Navaratnam V, Saini G, Thompson N, Booth HL, Hart SP, Hill MR, Hirani N, Maher TM, McAnulty RJ, Millar AB, Molyneaux PL, Parfrey H, Rassl DM, Whyte MKB, Fahy WA, Marshall RP, Oballa E, Bossé Y, Nickle DC, Sin DD, Timens W, Shrine N, Sayers I, Hall IP, Noth I, Schwartz DA, Tobin MD, Wain LV, Jenkins RG. Genetic variants associated with susceptibility to idiopathic pulmonary fibrosis in people of European ancestry: a genome-wide association study. Lancet Respir Med 2017, 5: 869-880. doi: https://doi.org/10.1016/S2213-2600(17)30387-9. PubMed PMID: 29066090.

5: Allen RJ,  Guillen-Guio B,  Oldham JM,  Ma SF,  Dressen A,  Paynton ML,  Kraven L,  Obeidat M,  Li X,  Ng M,  Braybrooke R, Molina-Molina M, Hobbs BD,  Putman RK,  Sakornsakolpat P,  Booth HL,  Fahy WA, Hart SP, Hill MR,  Hirani N,  Hubbard RB,  McAnulty RJ, Millar AB, Navaratnam V, Oballa E, Parfrey H, Saini G, Whyte MKB, Zhang Y, Kaminski N, Adengunsoye A, Sterk ME, Neighbors M, Sheng XR, Gudmundsson G, Gudnason V, Hatabu H, Lederer DJ, Manichaikul A, Newell Jr JD, O'Connor GT, Ortega VE, Xu H, Fingerlin TE, Bossé Y, Hao K, Joubert P, Nickle DC, Sin DD, Timens W, Furniss D, Morris AP, Zondervan K, Hall IP, Sayers I, Tobin MD, Maher TM, Cho MH, Hunninghake GM, Schwartz DA, Yaspan BL, Molyneaux PL, Flores C, Noth I, Jenkins RG, Wain LV. Genome-wide association study of susceptibility to idiopathic pulmonary fibrosis. Am J Respir Crit Care Med 2020, 201: 564-574. doi: https://doi.org/10.1164/rccm.201905-1017OC. PubMed PMID: 31710517.

6: Allen RJ, Stockwell A, Oldham JM, Guillen-Guio B, Flores C, Noth I, Yaspan BL, Jenkins RG, Wain LV, International IPF Genetics Consortium. Genome-wide association study across five cohorts identifies five novel loci associated with idiopathic pulmonary fibrosis. medRxiv. https://doi.org/10.1101/2021.12.06.21266509

